tests/IResearch/IResearchInvertedIndexConditionTest.cpp

```diff
@@ -100,6 +101,7 @@ class IResearchInvertedIndexConditionTest
       ASSERT_EQ(expected[i].coveringIndexPosition,
                 projections[i].coveringIndexPosition);
     }
+    ASSERT_TRUE(inverted->drop().ok());
```

Comment:
Could we make calling drop as some "finally"? Because if some of the ASSERTs above fails we could get failure due-to-missing-drop and mess the test results. Not critical I guess, as test will anyway do the job. Just want simple debugging for us later.

Reply:
Good point